lib/src/main/java/io/ably/lib/http/HttpUtils.java

```diff
@@ -63,4 +64,10 @@ public static String encodeURIComponent(String input) {
 		} catch (UnsupportedEncodingException e) {}
 		return null;
 	}
+
+	public static String getHeaderXAblyLib(String platform) {
+		return String.format("%s%s-%s", GlobalConstants.LIB_TYPE,
+										((platform == null) || (platform.length() == 0)) ? "" : "." + platform,
```

Comment:
Not sure if this is a Github issue, but tabbing looks very odd here, is it consistent with current tab?